docs/tutorials/services/constrain-motion.md

```diff
@@ -0,0 +1,410 @@
+---
+title: "Add Constraints and Transforms to a Motion Plan"
+linkTitle: "Add Motion Constraints"
+weight: 25
+type: "docs"
+description: "Use constraints and transforms with the Motion Service to move robot components in specific ways."
+webmSrc: "/tutorials/videos/motion_constraints.webm"
+mp4Src: "/tutorials/videos/motion_constraints.mp4"
+images: ["/tutorials/videos/motion_constraints.jpg"]
```

Comment:
This would be nicer as the gif of the video.

Reply:
Oh I thought we're supposed to also have this as a "poster" for previews? What is a poster?